### ISSUE TITLE
js2py fails when converting native object

### ISSUE THREAD

**Issue by jstoebel** [pybee/batavia#580]
when feeding a native javascript object into `types.js2py` I get the error:
```
/Users/stoebelj/batavia/batavia/types.js:156
                arg.__class__.__name__
                             ^

TypeError: Cannot read property '__name__' of undefined
    at Object.types.js2py (/Users/stoebelj/batavia/batavia/types.js:156:30)
    at Object.<anonymous> (/Users/stoebelj/spam.js:10:19)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:420:7)
```

I think this is caused by this conditional: ` arg.__class__ !== null && arg.__class__.__name__` In a native object, the first conditional will pass, but the second will throw an error because `arg.__class__` is `undefined` I'll look into this some more.

**Comment by abonie** [comment]
I encountered same problem and fixed it on my branch by changing first conditional to simply `arg.__class__` instead of `arg.__class__ !== null`. I had to take a break from working on batavia recently, so I did not submit this change in a PR.

It would be great, if you could make sure that my fix is correct or come up with something more suitable and make a PR :)

**Comment by jstoebel** [comment]
closed by #582
